bloomrun.js

```diff
@@ -16,6 +16,7 @@ function BloomRun (opts) {
   this._isDeep = opts && opts.indexing === 'depth'
   this._buckets = []
   this._properties = new Set()
+  this._defaultResult = null
```

Comment:
I would rather have the distinction between match all and catch all. Meaning regexs on their own should get a their own array and not the defaultResult which I would only see as a single object